tests/compiler/std/string.ts

```diff
@@ -527,6 +527,40 @@ assert("\uD801\uDC4D".toUpperCase() ==  "\uD801\uDC25",  "DESERET SMALL LETTER E
 assert("\uD801\uDC4E".toUpperCase() ==  "\uD801\uDC26",  "DESERET SMALL LETTER OI");
 assert("\uD801\uDC4F".toUpperCase() ==  "\uD801\uDC27",  "DESERET SMALL LETTER EW");
 
+assert("𐐘" == "𐑀".toUpperCase(), "v8 test 1");
+assert("ASSB" == "A\u00DFB".toUpperCase(), "v8 test 2");
+assert("AB" == "Ab".toUpperCase(), "v8 test 3");
+assert("AÜ" == "aü".toUpperCase(), "v8 test 4");
+assert("AÜ" == "AÜ".toUpperCase(), "v8 test 5");
+assert("ABCDEFGHIJ" == "ABCDEFGHIJ".toUpperCase(), "v8 test 6");
+assert("ABCDEFGHIJ" == "ABCDEFGHIj".toUpperCase(), "v8 test 7");
+assert("ABÇDEFGHIJ" == "ABÇDEFGHIj".toUpperCase(), "v8 test 8");
+assert("ABÇDEFGHIJ" == "ABçDEFGHIj".toUpperCase(), "v8 test 9");
+assert("ABCDEFGHIÁ" == "ABCDEfGHIÁ".toUpperCase(), "v8 test 10");
+assert("ABCDEFGHIÁ" == "ABCDEfGHIá".toUpperCase(), "v8 test 11");
+assert("AŸ" == "aÿ".toUpperCase(), "v8 test 12");
+assert("AΜ" == "aµ".toUpperCase(), "v8 test 13");
+assert("CSSBẶ" == "cßbặ".toUpperCase(), "v8 test 14");
+assert("FIFLFFIFFL" == "\uFB01\uFB02\uFB03\uFB04".toUpperCase(), "v8 test 15");
+assert("ABCÀCSSA" == "abcàcßa".toUpperCase(), "v8 test 16");
+assert("ABCDEFGHIÀCSSA" == "ABCDEFGHIàcßa".toUpperCase(), "v8 test 17");
+assert("ABCDEFGHIÀCSSA" == "abcdeFghiàcßa".toUpperCase(), "v8 test 18");
+
+// assert("ABCDE" == ("a" + "b" + "cde").toUpperCase(), "v8 test 19");
+assert("Α\u0301Ο\u0301Υ\u0301Ω\u0301" == "α\u0301ο\u0301υ\u0301ω\u0301".toUpperCase(), "v8 test 20");
+assert("\u{10400}\u{118A0}" == "\u{10428}\u{118C0}".toUpperCase(), "v8 test 21");
```

Comment:
wait! You still use it here =)

Reply:
What do you mean?

Reply:
Is it supported ""\u{10400}\u{118A0}""?

Reply:
Just to make sure, this issue is addressed. All the plane 1 characters are supported now.